docs/getting-started/getting-started.md

```diff
@@ -47,6 +47,36 @@ default and completely open source. You can follow along development on
   </em>
 </details>
 
+Once you've downloaded Upstream, start it like this:
+
+#### On Linux:
+
+1. make the package executable with: `chmod +x radicle-upstream-X.X.X.AppImage`
+2. start the app: `./radicle-upstream-X.X.X.AppImage`
+
+#### On macOS:
+
+Open the `radicle-upstream-x.x.x.dmg` package and install Upstream by dragging
+the `radicle-upstream` binary to the `/Applications` folder.
+
+![install Upstream on macOS][in]
+
+**Note**: before you can start Upstream for the first time, you'll have to
+perform the following steps. You'll only have to do this once, the first time
+you launch the app.
```

Comment:
Should we add an explainer of *why* they have to do this?

Reply:
How about this?

```
**Note**: before you can start Upstream for the first time, you'll have to
perform some extra steps. This is because we haven't set up Upstream to be
signed by Apple, read more about it [here][reasons]. You'll only have to do
this once, the first time you launch the app.
```

@cloudhead should we write down our political stance somewhere, so we can link it here?

Reply:
Could we perhaps say this instead as a stop-gap solution, so we can get this merged?

```
**Note**: before you can start Upstream for the first time, you'll have to
perform some extra steps. This is because we haven't set up
[Apple notarization][an] for Upstream yet, read more about it [here][aw].
You'll only have to do this once -- the first time you launch the app.

[an]: https://developer.apple.com/documentation/xcode/notarizing_macos_software_before_distribution
[aw]: https://support.apple.com/en-gb/guide/mac-help/mh40616/mac
```


Getting the signing working will take some time: https://github.com/radicle-dev/radicle-upstream/issues/1306.

Reply:
Let's do:

**Note**: before you can start Upstream for the first time, you'll have to
perform some extra steps. This is because we haven't set up
[Apple notarization][an] for Upstream yet, read more about it [here][aw].
You'll only have to do this once -- the first time you launch the app.

While we will be participating in the Apple Developer Program to ensure accessibility to our macOS users, we do not agree with Apple's financial gatekeeping and increasing control over application distribution on their platform. Forcing developers to "pay-to-play" under the guise of a security solution is an extractive form of vendor lock-in that inhibits the growth and accessibility of open software.

[an]: https://developer.apple.com/documentation/xcode/notarizing_macos_software_before_distribution
[aw]: https://support.apple.com/en-gb/guide/mac-help/mh40616/mac

Reply:
https://github.com/radicle-dev/radicle-docs/pull/26/commits/97bef89d7007ac6805aaab9aa5835fc8b292e107